library/std/src/memchr.rs

```diff
@@ -1,6 +1,9 @@
 // Original implementation taken from rust-memchr.
 // Copyright 2015 Andrew Gallant, bluss and Nicolas Koch
```

Comment:
This should be copied into the tests file as well.

Reply:
Done.